docs/BUILD.md

```diff
@@ -0,0 +1,449 @@
+* create long-running services that run in containers via [containerd](http://containerd.io)
+
+### Standard Packages
+
+EVE uses a number of standard linuxkit packages. These are enumerated below. Others may be added later.  
+
+**Live**
+
+* `init` packages
+    * `linuxkit/init` - for the `init` process
+    * `linuxkit/runc` - for `runc`
+    * `linuxkit/containerd` - for `containerd`
+    * `linuxkit/getty` - for `getty` to log in on console
+* `onboot` packages
+    * `linuxkit/sysctl`
+    * `linuxkit/modprobe` - customized for the appropriate modules
+    * `linuxkit/mount` - customized to mount the config partition as `/var/config`
+* `services` packages
+    * `linuxkit/openntpd` - for ntp
+
+**Installer**
+
+* `init` packages
+    * `linuxkit/init` - for the `init` process
+    * `linuxkit/runc` - for `runc`
+    * `linuxkit/getty` - for `getty` to log in on console
+
+### Custom Packages
+
+The remaining packages are custom images built for EVE. All of the packages - and some tools - are in the [pkg/](./pkg/) directory. We intend at some point to separate out tools, which are used at build-time, into the `tools/` directory, from packages, which are actual OCI images loaded into a runnable image, which will remain in `pkg/`.
+
+The following custom packages are used:
+
+**Live**
+
+* kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. _Why?_
+* `init` packages:
+    * `zededa/grub` - 
+    * `zededa/devices-trees` - 
+    * `zededa/fw` - 
+    * `zededa/xen` - 
+    * `zededa/gpt-tools` - 
+    * `zededa/dom0-ztools` - 
+* `onboot` packages:
+    * `zededa/rngd` - custom `zededa/rngd` package, rather than the standard linuxkit one. _Why?_
+* `services` packages:
+    * `zededa/wwan` - WWAN drivers and software. LTE? 3G? 2G?
+    * `zededa/wlan` - WLAN drivers and software.
+    * `zededa/guacd` - 
+    * `zededa/zedctr` - a "catch-all" package for EVE tools; see below.
+
+**Installer**
+
+* kernel: EVE uses its own custom kernel package, rather than one of the standard linuxkit ones. _Why?_
+* `init` packages:
+    * `zededa/grub` - 
+    * `zededa/devices-trees` - 
+    * `zededa/xen` - 
+    * `zededa/dom0-ztools` - 
+* `onboot` packages:
+    * `zededa/rngd` - custom EVE rngd package, rather than the standard linuxkit one. _Why?_
+    * `zededa/mkimage-raw-efi` - custom EVE version of `mkimage-raw-efi` to create an ext4 image, used to make the correct filesystems on the target install disk.
+
+#### zedctr
+
+The package `zededa/zedctr` is a "catch-all" package, composed of many different packages that would go into `services` separately. Its source is [pkg/zedctr](./pkg/zedctr), and is comprised of many different services.
+
+#### ztools
+
+The package `ztools` is the `go-provision` service that is responsible for managing the various components and deployments of a running EVE system. Its source is [go-provision](https://github.com/zededa/go-provision). 
```

Comment:
I'd say "services" instead of "service" here. I'll update the github name for the go-provision repo to match.

Reply:
Ah, there are multiple services there? Got it.

Reply:
Fixed